src/extension.ts

```diff
@@ -588,3 +620,33 @@ function openDocument(document_path: string) {
     window.showTextDocument(doc);
   });
 }
+
+function showAskSourceryQuickPick(recipes: Recipe[]) {
+  return new Promise((resolve) => {
+    const recipeNames = recipes.map((item) => item.name);
+    const recipeItems = recipes.map((item) => ({
+      label: item.name,
+      id: item.id,
+    }));
+
+    const quickPick = window.createQuickPick();
+    quickPick.placeholder = "Ask Sourcery.";
```

Comment:
Can you replace this with something more descriptive?
Something like "Select a recipe below, or type in anything you'd like to ask Sourcery here" (or anything better than that).

Reply:
Will do this in the next PR...